src/lib/Scenes/Favorites/Components/Categories/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import { View } from "react-native"
+
+class Categories extends React.Component<null, null> {
```

Comment:
@orta This is the placeholder component where the Categories tab view should go

Reply:
Rock